modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase

```diff
@@ -1008,6 +1008,127 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-11706. Verify that only one / is shown in the URLs inside the sitemap when Virtual Host is enabled"
```

Comment:
Verify that only one "/" is shown in the URLs inside the sitemap when Virtual Host is enabled.